sway/handlers.c

```diff
@@ -227,7 +227,15 @@ static void handle_view_destroyed(wlc_handle handle) {
 		locked_container_focus = false;
 		break;
 	}
-	set_focused_container(get_focused_view(&root_container));
+
+    swayc_t *focused_view = get_focused_view(&root_container);
```

Comment:
You'll want to alter the style for this. Spaces -> Tabs, and please add in spaces for if statements so they read like `if (cond) {`. Thanks!